public/firebase.js

```diff
@@ -196,8 +203,31 @@ class FirebaseIntegration {
     const userRef = firebase.firestore().collection('user').doc(userID);
     return firebase.firestore().collection(x).where(userFieldName, "==", userRef).get()
       .then((snapshot) => snapshot.docs.map((doc) => {
-        return {id: doc.id, data: doc.data()};
-      })
-    );
+        return {
+          id: doc.id,
+          data: doc.data()
+        };
+      }));
+  }
+
+  /**
+   * Internal method for deleting a document of a specific user
+   * @param x <string>
+   * @param collection <string>
+   * @param userFieldName <string>
+   * @param userID <string>
+   * @returns boolean
+   * @private
+   */
+  static deleteXFromUser(x, collection, userFieldName, userID) {
+
+    const userRef = firebase.firestore().collection('user').doc(userID);
+    firebase.firestore().collection(collection).doc(x).delete().then(function () {
+      console.log("Document successfully deleted!");
```

Comment:
Maybe this line could be removed because all other functions in firebase.js don't log to console.